middleware/allRoutes/handler.go

```diff
@@ -17,6 +17,8 @@ import (
 func Handler(routesHandler map[string]http.Handler) func(h http.Handler) http.Handler {
 	return func(h http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+			cfg, err := config.Get()
```

Comment:
Consider passing the cfg struct into this struct.

Reply:
Have updated to pass the cfg struct into the Handler function